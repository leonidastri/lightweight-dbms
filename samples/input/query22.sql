select * from Sailors S1, Boats B, Sailors S2 where S1.A = B.E order by S2.A;