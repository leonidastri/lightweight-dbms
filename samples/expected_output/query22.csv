2,200,200,101,2,3,1,200,50
2,200,200,107,2,8,1,200,50
3,100,105,102,3,4,1,200,50
1,200,50,103,1,1,1,200,50
1,200,50,103,1,1,2,200,200
2,200,200,101,2,3,2,200,200
2,200,200,107,2,8,2,200,200
3,100,105,102,3,4,2,200,200
1,200,50,103,1,1,3,100,105
2,200,200,101,2,3,3,100,105
2,200,200,107,2,8,3,100,105
3,100,105,102,3,4,3,100,105
2,200,200,107,2,8,4,100,50
3,100,105,102,3,4,4,100,50
1,200,50,103,1,1,4,100,50
2,200,200,101,2,3,4,100,50
1,200,50,103,1,1,5,100,500
2,200,200,101,2,3,5,100,500
2,200,200,107,2,8,5,100,500
3,100,105,102,3,4,5,100,500
2,200,200,101,2,3,6,300,400
2,200,200,107,2,8,6,300,400
3,100,105,102,3,4,6,300,400
1,200,50,103,1,1,6,300,400